runtime/j9vm/jvm.c

```diff
@@ -1492,6 +1495,7 @@ JITServer_CreateServer(JITServer **jitServer, void *serverArgs)
 	}
 	server->startJITServer = startJITServer;
 	server->waitForJITServerTermination = waitForJITServerTermination;
+	server->stopJITServer = stopJITServer;
```

Comment:
I don't see a call to this function occurring anywhere.  Should the jitserver launcher be updated to call it after waiting for `waitForJITServerTermination()` to free resources?
https://github.com/eclipse/openj9/blob/f2034099d7765870ecae55540642096e847fac52/runtime/jitserver_launcher/jitserver.c#L361

Reply:
Yes, jitserver would need to be updated to call `stopJITServer`, but we don't yet have any trigger to call this function. `waitForJITServerTermination` API should ideally be waiting for some 'event' which would tell it to call `stopJITServer`, but that we don't yet have that "event" in place.
As per the discussion in the parent issue, the mechanism to call `stopJITServer` needs to be discussed separately.

Reply:
Part of my problem with understanding the API is the rename from `JITServer_DestroyServer` -> `stopJITServer` as it confuses the purpose of the function and breaks the parallel with the `JITServer_CreateServer` call.

If this new function was changed back to `destoryJITServer` or similar name, the difference between it and `waitForJITServerTermination` would be clearer.  The expected pattern would then be something like:

```c
JITServer_CreateServer(&jitServer, ...);
....
jitServer-> waitForJITServerTermination();
//clean up resources
jitServer->destroyJITServer(&jitServer);
```

Reply:
ok, I don't mind changing the api from `stopJITServer` to `destroyJITServer`.